R/caretList.R

```diff
@@ -255,6 +255,18 @@ is.caretList <- function(object){
   is(object, "caretList")
 }
 
+#' @title Convert list of Caret Models to caretList object
+#' @description Convert list of Caret Models to caretList object
+#' @param modelList
+#' @export
+as.caretList <- function(modelList){
+  if(class(modelList) != "list"){
+    stop("modelList must be a list of caret models")
+  }
+  # TODO consider other parammeter checking perhaps check for the train class in the list??
```

Comment:
Let's add a check that all the objects in the list are of class train.  Something like `all(sapply(modelList, is.train))`